docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Global Description
+
+You would have a login page, a personal dashboard (number of images, etc).
+
+Homepage displays a list of users
+User's page displays a list of albums and in second time list of all pictures.
+If user doesn't have any album, just pictures, it would not display the album section.
+Album's page displays list of pictures belonging to the album
+
+# Uploader
+
+A user, when uploading can either populate an existing album or create a new one or just upload without album.
+There would be an uploader to upload multiple files at time with visual feedbacks (progression, error, success).
+Or you can create an album and then populate it.
+
+Images can be moved to an album later.
+
+Backend would create different images sizes optimized for web.
+
+# Album
+
+In an album, you have the list of pictures (thumbnail), album's properties. Album would have tags generated from all picture's tags inside it.
+
+Thumbnail of Album could be configured to be built based on random pictures inside the album or the last one or first one.
+
+Album can be private, kinda public (not listed but if you have the url you can access it), public.
+
+Album can be downloaded or not.
+
+## smart albums
+
+Albums that are generated automatically :
+- Uploads last 15 days
+- Favorites pictures (pictures could be faved)
+- Last batch uploaded
+- All public pictures
+
+# Pictures
+
+Pictures don't have a page strictly speaking, but there would be displayed full size in album when clicking on a thumbnail. You would a quite fullscreen with picture's properties info or just the image, two states would be toggled by clicking on the image.
+
+Pictures can have tags (that would be for a later use, maybe a search engine).
+
+# Preferences
+
+Per user you can have preferences for sorting, information display (exif or not, default privacy album, maybe image compression, possibility to download), then you can override it in album's preferences.
+
+One user must be elevated to admin, that would be the person that would install Poster obviously. The person could choose to have like just a personal gallery (1 user = admin) or a shared hosting gallery (N users + 1 admin).
+If just one user, the homepage would be the user's album listing page.
+
+There would be a dark and light theme.
+
+# Menu
```

Comment:
Any thoughts on a `duplicate`album button ?

Reply:
That would make sense if you would like to make an altered album public (from private) ?

Reply:
Yes or to start an album with some images in it